packages/node_modules/node-red/lib/red.js

```diff
@@ -25,11 +25,9 @@ var api = require("@node-red/editor-api");
 var server = null;
 var apiEnabled = false;
 
-const NODE_MAJOR_VERSION = process.versions.node.split('.')[0];
-if (NODE_MAJOR_VERSION >= 16) {
-    const dns = require('dns');
-    dns.setDefaultResultOrder('ipv4first');
-}
+// Workaround for WebSocket: https://github.com/node-red/node-red/issues/4010
```

Comment:
```suggestion
// Ensure ipv4 results are returned first: https://github.com/node-red/node-red/issues/4010
```

Small update to the comment as this is not specific to websockets